doc/tests/manual/reports/231121.md

```diff
@@ -0,0 +1,85 @@
+# Manual Test Report (v0.3.6)
+
+## General
+
+- Tester: Aydan Namdar Ghazani
+- Date/Time: 21. November 2023
+- Duration: 5 minutes
+- Commit/Tag: [671cab1](https://github.com/ElektraInitiative/PermaplanT/pull/1062/commits/671cab1866fc94f24202892b01dc96e571558429)
+- Setup: localhost
+- Planned tests: **6**
+- Executed tests: **6**
+- ✔️Passed tests: **6**
+- ⚠️Problematic tests: **0**
+- ❌Failed tests: **0**
+
+## [Error Analysis](../README.md#report-header)
+
+"Sorry I couldnt load your image" error occured on opening a fresh map.
```

Comment:
I thought this was something known/unfixed yet. Because there is also many errors in the consoles about images not beeing loaded. I cant reproduce it anymore, but in very rare situations I get this error.

Reply:
If you manage to reproduce it, please report. There shouldn't be any weird things anymore, we are starting to get  production-ready.